web/src/components/itemList.js

```diff
@@ -29,17 +29,22 @@ class ItemList extends Component {
         this.rate = 0.001;
     }
 
-    itemClicked = event => {
+    addItemClicked = event => {
+        const idx = event.target.getAttribute('id');
+        this.selected[idx]++;
+        this.props.addItems(this.state.items[idx].price, idx);
+        event.target.setAttribute('variant', 'primary')
+    }
+
+    removeItemClicked = event => {
         const idx = event.target.getAttribute('id');
         if(this.selected[idx]) {
-            this.props.removeItem(this.state.items[idx].price);
+            this.selected[idx]--;
+            this.props.removeItem(this.state.items[idx].price, idx);
             event.target.setAttribute('variant', 'primary')
```

Comment:
This is not working. To understand why this is not working, you need to understand about lifecycle of react.js component.
This issue is related to line 81.